### PR TITLE
Add OpenSky Network API and remove Salesfly PDF API

### PR DESCRIPTION
**Pull request by arthurlli**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by arthurlli** [comment]
I found url for Salesfly PDF API is invalid, so I modified and attached another link, which is squashed with my previous commit. Hopes this helps.

File: README.md [hunk 1]
```diff
@@ -377,7 +377,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Pocket](https://getpocket.com/developer/) | Bookmarking service | `OAuth` | Yes | Unknown |
 | [PrexView](https://prexview.com) | Data from XML or JSON to PDF, HTML or Image | `apiKey` | Yes | Unknown |
 | [Restpack](https://restpack.io/) | Provides screenshot, HTML to PDF and content extraction APIs | `apiKey` | Yes | Unknown |
-| [Salesfly PDF](https://salesflypdf.com) | HTML/URL to PDF | `apiKey` | Yes | Yes |
+| [Salesfly PDF](https://developers.salesfly.com/swift/) | HTML/URL to PDF | `apiKey` | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:380]: This API is in private beta.


image

Can you remove, please?

```suggestion
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Hi @arthurlli ✋ 

Thanks for your contribution, we are grateful for that ❤️ 

Can you resolve the review, please?

**Comment by arthurlli** [comment]
Hi @matheusfelipeog, thank you for the review, I have updated and squashed the commits, please check. Thanks!:smile:

**Comment by arthurlli** [comment]
> 
> 
> Hi @matheusfelipeog, thank you for the review, I have updated and squashed the commits, please check. Thanks!😄

I just noticed that I "force-push" the commit, sorry if it may cause some issue.

**Review by matheusfelipeog — APPROVED** [review]
Perfect ❤️

**Comment by matheusfelipeog** [comment]
> I just noticed that I "force-push" the commit, sorry if it may cause some issue.

No problem, you did the right. 😆

**Comment by arthurlli** [comment]
Great! Thank you!

**Review by matias904 — CHANGES_REQUESTED** [review]
- [ ]

**Pull request merged**